.github/workflows/ios-test.yml

```diff
@@ -24,8 +24,17 @@ jobs:
                   node-version: 16
                   cache: "npm"
                   cache-dependency-path: "**/package-lock.json"
+            - name: Cache multiple paths
+              uses: actions/cache@v2
+              id: cache
+              with:
+                  path: "**/node_modules"
+                  key: ${{ runner.os }}-${{ hashFiles('**/package-lock.json') }}
```

Comment:
Apparently only cypress uses ~/.cache, and that's only used in tests, which is then not needed here.

> We could argue whethere we *actually* need to install cypress, but, well...